testkit/src/main/java/org/apache/calcite/test/RelOptFixture.java

```diff
@@ -76,8 +76,10 @@
  * <p>A fixture is immutable. If you have two test cases that require a similar
  * set up (for example, the same SQL expression and set of planner rules), it is
  * safe to use the same fixture object as a starting point for both tests.
+ *
+ * <p>The class has public access-level to allow implementing custom fixtures.
```

Comment:
I don't think this paragraph adds much. I think you should remove it. (It's not even true; the constructor remains package-private because we don't want people subclassing.)